spec/features/purchases_spec.rb

```diff
@@ -30,6 +33,33 @@
     expect(numrows).to be == 5
   end
 
+  scenario 'display three purchases after clicking on week button' do
+    click_link 'Purchases'
+    click_on 'button_Week'
+
+    numrows = page.all(:css, 'table tbody tr').size # subtracting header row
```

Comment:
Remove comment about subtracting header row (also below)

Reply:
Also, `numrows` isn't a very Rubyish name. Actually, you could simply call the var `purchases`